packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1232,12 +1228,6 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 	});
 
 	it("load offline with blob redirect table", async function () {
-		// TODO:AB#4746: Resolve flakiness of this test on CI.
```

Comment:
I have tested this locally 1000 times. Blob manager has changed a lot as well so it should be more stable now

Reply:
I know it can still fail on CI but I believe it's fair to add it back for now and check if it fails again